docs/src/examples/components/Input/Variants/Variants.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react'
```

Comment:
these temporary changes suggest the way how variants could be rendered to docs in automated way - won't be merged